rgw/admin/link.go

```diff
@@ -5,16 +5,15 @@ import (
 	"net/http"
 )
 
-// BucketLinkInput the bucket link/unlink input parameters
-type BucketLinkInput struct {
-	Bucket   string `url:"bucket" json:"bucket"`
-	BucketID string `url:"bucket-id" json:"bucket_id"`
-	UID      string `url:"uid" json:"uid"`
```

Comment:
Unless an empty value for "new-bucket-name" is problematic, I think it should be fine to add `NewBucketName` to the existing `BucketLinkInput` struct and the diff looks much cleaner.

Reply:
After stating my opinion, please let me know your decision so we can continue on this topic.

Reply:
> After stating my opinion, please let me know your decision so we can continue on this topic.

Sorry, I was away for the last two weeks. Let's reuse the existing `BucketLinkInput` struct for now.

Reply:
Ah sorry, I was just afraid it would be forgotten. 
I've adjusted the PR accordingly